tests/plugins/imagebase/features/manual/uploadsimple.html

```diff
@@ -3,7 +3,7 @@
 	<p></p>
 </div>
 
-<script src="../../../easyimage/manual/_helpers/tools.js"></script>
+<script src="../../../easyimage/_helpers/tools.js"></script>
```

Comment:
Change made because of change location of `CLOUD_SERVICES_UPLOAD_GATEWAY` which is used below.